provider/terraform.rb

```diff
@@ -165,7 +165,7 @@ def generate_resource_tests(data)
     def generate_operation(output_folder, _types, version_name)
       return if @api.objects.select(&:autogen_async).empty?
 
-      product_name = @api.name.underscore
+      product_name = @api.name.camelize(:lower)
```

Comment:
I think `underscore` was correct here; we're using this var for the filename where we want `resource_manager_operation.go` and not `resourceManager_operation.go`.

Reply:
You're completely right!